### ISSUE TITLE
serious question 

### ISSUE THREAD

**Issue by Silentsniper1** [UnamSanctam/UnamBinder#63]
How can the binder be effective can u give an example or help me because im not actually understanding it purpose please and thank u

**Comment by Silentsniper1** [comment]
okay can i bind the Exe(miner) file for example to a video or a photo so when someone clicks it the miner automatically encrypts it self or no?

**Comment by UnamSanctam** [comment]
Both the miner and the photo/video will be encrypted inside the bound file, then when the bound file is run it decrypts the miner and photo/video and runs both of them.

**Comment by Silentsniper1** [comment]
and what do you mean by bound file (sorry for asking alot of questions)

**Comment by UnamSanctam** [comment]
Yes when the file you create with UnamBinder (the bound file) is run then it will start both the miner and the picture/video.
